src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/ProjectInfoParams.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+
+internal class ProjectInfoParams
+{
+    public required string ProjectKeyId { get; set; }
+
+    public required string? ProjectInfo { get; set; }
```

Comment:
Not for this PR, but we should probably follow up in a short-ish time frame to make this proper serialization. Could just put the right JSON attributes on the `RazorProjectInfo` class. Right now this is going to be an LOH allocation with every call I think.

Reply:
I'll resolve for this PR and will clean up shortly after so this PR doesn't linger any longer than necessary :)